docs/UserGuide.md

```diff
@@ -664,7 +665,7 @@ Example:
 
 :information_source: **Information:**
 
-* If there are no undo commands executed previously, the redo command will fail and an error message will be shown.
+* If there are no undoable commands executed previously, the redo command will fail and an error message will be shown.
```

Comment:
This one should be 'undo'? The `redo` command will only work if the user has previously undone something.